pkg/config/config.go

```diff
@@ -560,3 +572,42 @@ func validateAuditLogConfig(cfg AuditLog) error {
 	}
 	return errors.Join(errs...) // Join returns nil if len(errs) == 0
 }
+
+func validateNetworkStack(cfg *Config) error {
+	if len(cfg.Network.ClusterNetwork) != len(cfg.Network.ServiceNetwork) {
+		return fmt.Errorf("network.serviceNetwork and network.clusterNetwork have different cardinality")
+	}
+	if len(cfg.Network.ServiceNetwork) > 2 {
+		return fmt.Errorf("network.serviceNetwork can not have more than 2 entries")
+	}
+	if len(cfg.Network.ClusterNetwork) > 2 {
+		return fmt.Errorf("network.clusterNetwork can not have more than 2 entries")
```

Comment:
This will never happen. If both slices needs to be same length, only `if` above will effectively work

Reply:
you are right. done.